front/src/pages/_app.js

```diff
@@ -1,11 +1,16 @@
 import "../../styles/globals.css";
-import { PageWrapper } from "../components/page-wrapper/index";
+import {PageWrapper} from "../components/page-wrapper/index";
+import {QueryClientProvider, QueryClient} from "react-query";
 
-function MyApp({ Component, pageProps }) {
+const queryClient = new QueryClient();
```

Comment:
We should think whether we want to instantiate `queryClient `here or within hooks where we will make the requests. It's okay for now, just a topic for us to think about.